api-reference/10 UI Widgets/dxSortable/3 Methods/update().md

```diff
@@ -3,7 +3,18 @@ id: dxSortable.update()
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Updates **Sortable**'s dimensions. Call this method after items are added or their dimensions are changed during dragging.
```

Comment:
Is "Sortable" the method name?
Can you clarify how this method works with dimensions, for my understanding?

Reply:
Sortable is a widget. Here's the demo;
https://js.devexpress.com/Demos/WidgetsGallery/Demo/Sortable/Kanban/Vue/Light/
In that demo, the dimensions are the layout of one column and its inner elements.

Reply:
Thank you. So, when I drag a column to a different position, the size (i.e., dimensions) seem to stay the same. Can you clarify how the dimension changes?